cache/manager_test.go

```diff
@@ -1985,6 +1998,15 @@ func checkInfo(ctx context.Context, t *testing.T, cs content.Store, info content
 	if !ok {
 		return
 	}
+
+	desc, err := getBlobDesc(ctx, cs, info.Digest)
+	require.NoError(t, err)
+	if isNydusBlob(desc) {
+		// Nydus compression type hasn't yet implemented decompression,
+		// just skip here.
+		return
+	}
```

Comment:
How can we ensure the created nydus is valid and preserves the contents of the original tar image?